source/draw/gpu/pc/Window.ooc

```diff
@@ -23,7 +23,7 @@ use ooc-base
 import X11/X11Window
 import X11/include/x11
 
-Window: class extends GpuSurface {
+Window: class extends OpenGLSurface {
```

Comment:
Kind of, I have another PR which sorts out the Window mess where this class will be named OpenGLWindow instead and be a part of the OpenGL module.